### DISCUSSION TITLE
Shortcut to one of the Control button methods

### DISCUSSION THREAD

**Discussion by 0xtf** [HomeSpan/HomeSpan#573] [Q&A] [answered]
Hi there,

I was wondering what would be the easiest way to "shortcut" into one of the 5 methods available under the control button options. My goal is to use the control button with one of the methods pre-configured, without the option to cycle through them. 

Thanks!

**Comment by HomeSpan** [comment, marked as answer]
The easiest would probably be to use a dedicated button (instead of, or in addition to, the control button) and call the desired function directly using the `homeSpan.processSerialCommand(const char *CLIcommand)` method.

**Reply by 0xtf** [reply]
> The easiest would probably be to use a dedicated button (instead of, or in addition to, the control button) and call the desired function directly using the `homeSpan.processSerialCommand(const char *CLIcommand)` method.

That's great! Thank you!
